.gitmodules

```diff
@@ -1,6 +1,6 @@
-[submodule "external/pybind11"]
-	path = bindings/python/external/pybind11
-	url = https://github.com/pybind/pybind11
+[submodule "bindings/python/external/nanobind"]
+	path = bindings/python/external/nanobind
+	url = https://github.com/ManifoldFR/nanobind
```

Comment:
please point to the official repo if possible :-)

Reply:
Done. I think by the way the fact we are using the submodule kind of reverts some of the work you have been doing @nim65s ?